alien4cloud-yorc-plugin/src/main/resources/google/resources/resources.yaml

```diff
@@ -0,0 +1,132 @@
+tosca_definitions_version: ${alien4cloud.dsl.version}
+
+template_name: yorc-google-types
+template_author: Yorc
+template_version: ${yorc.types.version}
```

Comment:
Should use a dedicated version for GCP

Reply:
Ok, seeing this change now in develop branch, will update